figshare/figshare.py

```diff
@@ -1,8 +1,4 @@
-import sys
-import os
-import hashlib
 import json
-
```

Comment:
Let's keep this newline but delete the gap between `import requests` and `from requests.exceptions import HTTPError`. As per PEP8